docs/features/OxHugo compatibility.md

```diff
@@ -0,0 +1,36 @@
+---
+tags:
+  - plugin/transformer
+---
+
+Quartz is a static-site generator that transforms markdown content into web pages. [org-roam](https://www.orgroam.com/) is a plain-text(`org`) personal knowledge management system for [emacs](https://en.wikipedia.org/wiki/Emacs). [ox-hugo](https://github.com/kaushalmodi/ox-hugo) is org exporter backend that exports `org-mode` files to [Hugo](https://gohugo.io/) compatible markdown.
+
+Because the markdown generated by ox-hugo is not pure markdown but hugo specific, we need to transform it to fit into Quartz. This is done by [OxHugoFlavouredMarkdown](https://github.com/jackyzha0/quartz/blob/v4/quartz/plugins/transformers/oxhugofm.ts). Even though this [[plugin]] was written with `ox-hugo` in mind, it should work for any hugo specific markdown. Please feel free to add any changes that you feel necessary.
+
+```typescript
+plugins: {
+  transformers: [
+    Plugin.FrontMatter({ delims: "+++", language: "toml" }), // if toml frontmatter
+    // ...
+    Plugin.OxHugoFlavouredMarkdown(),
+    Plugin.GitHubFlavoredMarkdown(),
+    // ...
+  ],
+},
+```
+
+## Usage
+
+Quartz will not try to export your org files into markdown, since the `org-roam` ecosystem is external you're responsible for providing the markdown content to quartz and then managing the static assets so that they're available in the final output.
+
+The way this is being used right now is not super convenient but illustrated below(using makefile) but you could write a [[making plugins|custom emitter plugin]] if you wanted.
+
+```mermaid
+graph LR
+  org-roam-- ox-hugo export -->hcm[hugo-compatible markdown] --> ctq[copy contents to quartz repo] --> qb[quartz build] --> cp[contents in public directory]
+```
+
+```mermaid
+graph LR
+  org-roam-- ox-hugo export -->hcm[hugo-static assets] --> qb[copy to public directory] --> w[push to web]
+```
```

Comment:
ok to omit this

Reply:
please also add a `## Configuration` heading that contains information about the available options (see docs pages on other transformers for more info). You should also create a warning callout to mention that this shouldn't be enabled at the same time as the `ObsidianFlavoredMarkdown` plugin